po/ja.po

```diff
@@ -773,33 +740,28 @@ msgid "Changing API"
 msgstr "APIの変更"
 
 #: src/SUMMARY.md
-#, fuzzy
 msgid "Updating Implementations"
-msgstr "実装"
+msgstr "実装の更新"
 
 #: src/SUMMARY.md
-#, fuzzy
 msgid "AIDL Types"
-msgstr "型"
+msgstr "AIDL型"
 
 #: src/SUMMARY.md src/android/aidl/types/primitives.md
 msgid "Primitive Types"
 msgstr ""
 
 #: src/SUMMARY.md src/android/aidl/types/arrays.md
-#, fuzzy
 msgid "Array Types"
-msgstr "配列"
+msgstr "配列型"
 
 #: src/SUMMARY.md src/android/aidl/types/objects.md
-#, fuzzy
 msgid "Sending Objects"
-msgstr "トレイトオブジェクト"
+msgstr "オブジェクトの送信"
 
 #: src/SUMMARY.md src/android/aidl/types/parcelables.md
-#, fuzzy
 msgid "Parcelables"
-msgstr "変数"
+msgstr "パーセラブル"
```

Comment:
```suggestion
msgstr "Parcelables"
```

I think this is a technical term that we don't have no Japanese word. So, I guess keeping it in English would be more helpful
https://developer.android.com/reference/android/os/Parcelable